src/index.html

```diff
@@ -48,4 +48,4 @@
   
   </body>
 
-</html>
\ No newline at end of file
+</html>
```

Comment:
you might want to install Editorconfig, it looks like you've left a bunch of newlines at the end of all these files.  Editorconfig is configured not to add them.